README.md

```diff
@@ -15,9 +15,9 @@ English | [简体中文](./README-zh_CN.md)
 
 dt-sql-parser is a **SQL Parser** project built with [ANTLR4](https://github.com/antlr/antlr4), and it's mainly for the **BigData** domain. The [ANTLR4](https://github.com/antlr/antlr4) generated the basic Parser, Visitor, and Listener, so it's easy to complete the **syntax validation**, **tokenizer**, **traverse** the AST, and so on features.
 
-Besides, it provides some helper methods, like **split** SQL, and filter the `--` and `/**/` types of comments in SQL.
+Besides, it provides some helper methods, like **split** SQL, and **Auto-Complete**.
```

Comment:
Additionally, it provides auxiliary functions such as SQL splitting and Auto-Complete.